src/components/Accordion/AccordionItem/AccordionItem.jsx

```diff
@@ -7,28 +7,33 @@ import {AccordionContext} from '~/components/Accordion/Accordion.context';
 
 export const AccordionItem = ({id, label, icon, onClick, children, className, ...props}) => {
     const context = useContext(AccordionContext);
+    const [cssExpanded, setCssExpanded] = useState(null);
     const open = context.currentItem === id;
 
     const handleClick = (e, open) => {
         onClick(e, !open);
         context.onSetOpenedItem(id);
     };
 
+    useEffect(() => {
+        setCssExpanded({[styles.accordionItem_opened]: open});
+    },
+    [open]);
+
```

Comment:
If only for the first render I would suggest something like that : 
```suggestion
    const [initialized, setInitialized] = useState(false);
    useEffect(() => {
        setInitialized(true)
    }, []);
    const isExpanded = open && initialized
```
And replace the cssExpanded  with just `{[styles.expanded]: isExpanded}`